prettier.config.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * See https://prettier.io/docs/en/options.html.
+ */
+module.exports = {
+  printWidth: 80,
+  semi: true,
+  singleQuote: true,
+  jsxSingleQuote: false,
+  quoteProps: 'consistent',
+  trailingComma: 'all',
```

Comment:
will this cause any non-compiled JS to break?
I am all for this setting but just wanted to make sure we do not cause issues with JS that does not pass through a transpiler.

https://prettier.io/docs/en/options.html#trailing-commas

Reply:
This was a concern while we were supporting IE11, but now that we aren’t, all our target browsers will support this syntax (see the row at https://kangax.github.io/compat-table/es2016plus/#test-trailing_commas_in_function_syntax).